tests/test_read_only.py

```diff
@@ -93,7 +93,9 @@ def get_container(
     container = None
     try:
         container = client.containers.run(
-            image=os.getenv("DOCKER_CAPELLA_READONLY", "capella/readonly"),
+            image=os.getenv("DOCKER_PREFIX", "")
+            + "capella/readonly:"
+            + os.getenv("DOCKER_TAG", "latest"),
```

Comment:
It's more readable if we use an f-string here.

Reply:
I can do: 
```
image=(
                f"{os.getenv('DOCKER_PREFIX', '')}capella/readonly"
                f":{os.getenv('DOCKER_TAG', 'latest')}"
            ),
```

Otherwise, the line gets too long.

Reply:
What do you think?

Reply:
@amolenaar Did you have some time to look into it?

Reply:
@MoritzWeber0 Splitting a line like that (two strings on separate lines) does work in python, but it doesn't make it more readable. Then better stick with good old string concatenation.

Alternatively, create a variable.

Reply:
I've created two new variables `docker_prefix` and `docker_tag`. This makes it a lot more readable.